CodePush.js

```diff
@@ -575,17 +574,17 @@ function codePushify(options = {}) {
       render() {
         const props = {...this.props};
 
-        // we can set ref property on class components only (not stateless)
-        // check it by render method
+        // We can set ref property on class components only (not stateless)
+        // Check it by render method
         if (RootComponent.prototype.render) {
```

Comment:
@MikhailSuendukov  Does it make sense to add check for `RootComponent.prototype` as mentioned in https://github.com/microsoft/react-native-code-push/issues/2037#issuecomment-1006269334 ?

Reply:
Done